MobileFilter/sections/specific_web.txt

```diff
@@ -43,6 +43,7 @@ vg247.com##div[id^="mobile-mpu-"]
 woman.ru##.mob-in__container
 woman.ru##.notification__ad
 woman.ru##div[data-name="MobListVideo"]
+metrolyrics.com###videoContainer
```

Comment:
I meant with this kind, so it looks like this in Annoyances filter:
!+ PLATFORM(android, ios, ext_android_cb)
metrolyrics.com###videoContainer

Reply:
I couldn't find this string in Annoyances filter, just !+ PLATFORM(ios, ext_android_cb). 
Can you show where it should be?

Reply:
When you add !+ PLATFORM(android, ios, ext_android_cb) one line above the rule, it tells the compiler to only put the rule into filter lists for the mentioned platforms.

So it should like this in Annoyances filter:

eva.ru###subscribe-dialog
navalny.com###pushPopup
!+ PLATFORM(android, ios, ext_android_cb)
metrolyrics.com###videoContainer